README.md

```diff
@@ -169,6 +169,8 @@ A VM can be tuned at two different places with the following keys:
 - `root_password`: the root password in clear text
 - `ssh_key_file`: the path of the public key for connecting to the VM
 - `groups`: this list of groups will be used if you generate an Ansible inventory.
+- `disks`: a list of additional disks to create and attach to the VM.
```

Comment:
I don't like the fact we've got both `disks` and `root_disk_size`. I would prefer that if we use `disks`, the `size` field of the first entry is be used to initialize `root_disk_size`.

We should also raise an error if `root_disk_size` and `disks` are both in use.